pennylane/operation.py

```diff
@@ -531,6 +531,10 @@ def parameters(self):
         """Current parameter values."""
         return self.data.copy()
 
+    def decomposition(self):
+        """Returns a template decomposing the operator into other operators"""
```

Comment:
```suggestion
        """Defines a decomposition of this operator into products of other operators
        
        Returns:
            list[Operation]  
        """
```

(check indents)

This will be clearer for developers overwriting the method I hope?